index.rst

```diff
@@ -492,14 +492,21 @@ What's new in PyTorch tutorials?
    :image: _static/img/thumbnails/cropped/60-min-blitz.png
    :link: advanced/static_quantization_tutorial.html
    :tags: Quantization
-   
+
 .. customcarditem::
    :header: Grokking PyTorch Intel CPU Performance from First Principles
    :card_description: A case study on the TorchServe inference framework optimized with Intel® Extension for PyTorch.
    :image: _static/img/thumbnails/cropped/generic-pytorch-logo.png
    :link: intermediate/torchserve_with_ipex
    :tags: Model-Optimization,Production
 
+.. customcarditem::
+   :header: Multi-Objective Neural Architechture Search with Ax
```

Comment:
nit: Architecture*

Reply:
Architechture is a cool word though in the context of NAS
```suggestion
   :header: Multi-Objective Neural Architecture Search with Ax
```